docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
+- [ ] Large-scale scene animations (e.g., pedestrian simulations, weather animations, disaster simulations).
+- [ ] Support atmospheric rendering effects.
 
 ### Extensions
 
-- Enhance publishing workflow to Bentley's Extension Service.
-- Create examples and templates for building and publishing Extensions.
-- Support for better testing in the Extension's Hosting App, i.e. iTwin Viewer, iTwin Design Review, PlantSight, etc.
+- [ ] Enhance publishing workflow to Bentley's Extension Service.
+- [ ] Create examples and templates for building and publishing Extensions.
+- [ ] Support for better testing in the Extension's Hosting App, i.e. iTwin Viewer, iTwin Design Review, PlantSight, etc.
 
 ### GIS Display
 
-- Expand base map support to include additional sources (AzureMaps, MapBox etc.) or solid color. Transparency of base map can be controlled independently of layers.
-- Add support for display of raster map layers connected to live WMS, WMTS, AzureMaps, MapBox, or ArcGIS REST API servers. Map layer visibility and transparency can be controlled independently. Layers can be attached either as background below BIM geometry or overlays above BIM geometry.
+- [X] Expand base map support to include additional sources (AzureMaps, MapBox etc.) or solid color. Transparency of base map can be controlled independently of layers.
+- [X] Add support for display of raster map layers connected to live WMS, WMTS, AzureMaps, MapBox, or ArcGIS REST API servers. Map layer visibility and transparency can be controlled independently. Layers can be attached either as background below BIM geometry or overlays above BIM geometry.
 
 ### Microsoft Teams integration
 
-- Provide api for hosting iTwin.js applications directly within Microsoft Teams.
+- [ ] Provide api for hosting iTwin.js applications directly within Microsoft Teams.
 
 ### ET/IT/OT Integration
 
-- Provide working examples and templates for interfacing between engineering content in iModels, enterprise systems, and realtime IOT sensors, cameras, controllers, devices, and processors.
-- Create tools for augmenting data synchronized with connectors from engineering design tools with IOT-link elements.
-- Enhance BIS schemas for common IOT patterns and query engines.
+- [ ] Provide working examples and templates for interfacing between engineering content in iModels, enterprise systems, and realtime IOT sensors, cameras, controllers, devices, and processors.
+- [ ] Create tools for augmenting data synchronized with connectors from engineering design tools with IOT-link elements.
+- [ ] Enhance BIS schemas for common IOT patterns and query engines.
 
 ### Agent Deployment
 
-- Provide working examples and templates of agents deployed via Kubernetes.
-- Perform operations on iModels that are triggered by relevant events from iModelHub.
+- [ ] Provide working examples and templates of agents deployed via Kubernetes.
+- [ ] Perform operations on iModels that are triggered by relevant events from iModelHub.
```

Comment:
@wgoehrig @ramanujam-raman this would apply to agents like tile generation and version comparison?

Reply:
In absence of a response I have marked these to the best of my knowledge.

Reply:
I reverted your change to the iModel triggered events support as this isn't fully working within the iTwin Platform APIs. That's still a work in progress.